### ISSUE TITLE
Cache totals data to reduce processing time

### ISSUE THREAD

**Issue by jakcodex** [jakcodex/muledump#127]
Users with a large number of items or on slower machines currently experience page responsiveness issues whenever totals are calculated. Presently, this happens every time there's an account change, screen change, every reload, and other times.

Totals should only be calculated if its specific options are modified or if account data changes. It can be loaded from cache any other time to reduce cpu usage.

**Comment by jakcodex** [comment]
Added new class `Muledump_TotalsCounter`.

```js
var global = new Muledump_TotalsCounter('global');
var counter = new Muledump_TotalsCounter(guid);
var itemId = 2397;
var quantity = 33;
counter.count(itemId, quantity);
```

Aggregate totals are stored in ` Muledump_TotalsCounter.data.totals`.

Total types are stored in ` Muledump_TotalsCounter.data.types`.

For the purposes of keeping a global totals list, an import method is made available to load data from multiple trackers. The global totals list gets regenerated any time a Mule.parse() is called and its totals prepares to save the data cache.

Mule tracker cache data is invalidated any time Mule.parse() is called.

**Comment by jakcodex** [comment]
Added in options to import() for excluding a list of guids or resetting the exclusion list.

Global Totals will now only calculate for active accounts that aren't disabled (via ctrl+click), excluded, or missing from accounts filter.

**Comment by jakcodex** [comment]
Published in v9.3.0.
